src/modules/jcms_article/src/Entity/ArticleVersions.php

```diff
@@ -39,7 +39,7 @@
    * @param string $json
    * @param int $action
    */
-  public function __construct(string $id, string $json, int $action = self::WRITE) {
+  public function __construct(string $id, string $json = '', int $action = self::WRITE) {
```

Comment:
This allows an article node to be created by just supplying an article id which is utilised on migrate.